python-scripts/read_rss_default_analysis.py

```diff
@@ -1,49 +1,73 @@
 """
-read_rss_default_analysis.py
+read_rss.py
 
-A simple rss reader in Python that does sentiment analysis and puts the built_in_sias in Deephaven
+A simple RSS reader with pulling logic
```

Comment:
read_rss_default_analysis.py

A simple rss reader in Python that does sentiment analysis using the NLKT default sentiment analysis, and stores the results in Deephaven